determined-ai/determined
# feat: conditionally add genai to the sidebar

## Description





## Test Plan
- enable the feature switch `GAS`
- run the web ui against it it should show a sidebar nav item

![Screenshot 2023-12-07 at 17 38 16](https://github.com/determined-ai/determined/assets/12127420/864c7ce0-b18e-4a71-9748-7a259f734544)






## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket
